ifrs17/Import/ImportStorage.ipynb

```diff
@@ -477,6 +479,15 @@
                 "\n    public IEnumerable<string> GetClaims() => hierarchyCache.Get<AmountType>(AmountTypes.CL).Descendants().Select(x => x.SystemName);",
                 "\n    public IEnumerable<string> GetCdr() => hierarchyCache.Get<AmountType>(AmountTypes.CDR).Descendants(includeSelf: true).Select(x => x.SystemName);",
                 "\n    public IEnumerable<string> GetCoverageUnits() => hierarchyCache.Get<AmountType>(AmountTypes.CU).Descendants(includeSelf: true).Select(x => x.SystemName);",
+                "\n",
+                "\n    private YieldCurve LogNoYcError(string dn, bool locked) =>",
```

Comment:
Remove the LogNoYcError method. 
- it is used only twice, 
- taking out the DataNodeValidation(see next comment) is pretty simple
- the bool makes it not nice to read.

Reply:
done